manifests/cf-manifest/scripts/generate-cf-secrets.rb

```diff
@@ -14,6 +14,7 @@
   "bulk_api_password" => :simple,
   "nats_password" => :simple,
   "router_password" => :simple,
+  "route_services_secret" => :simple,
```

Comment:
We've done some digging, and I think this password scheme is fine. I've added details of this to the commit message.